app/src/debug/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,24 @@
+<resources>
+    <!--
+    TODO: Before you run your application, you need a Google Maps API key.
+
+    To get one, follow this link, follow the directions and press "Create" at the end:
+
+    https://console.developers.google.com/flows/enableapi?apiid=maps_android_backend&keyType=CLIENT_SIDE_ANDROID&r=39:18:DC:C4:4C:6F:2A:B3:24:8F:EA:DC:96:EF:5D:94:0D:C6:23:1E%3Bcom.metro.gwuexplorer
+
+    You can also add your credentials to an existing key, using these values:
+
+    Package name:
+    39:18:DC:C4:4C:6F:2A:B3:24:8F:EA:DC:96:EF:5D:94:0D:C6:23:1E
+
+    SHA-1 certificate fingerprint:
+    39:18:DC:C4:4C:6F:2A:B3:24:8F:EA:DC:96:EF:5D:94:0D:C6:23:1E
+
+    Alternatively, follow the directions here:
+    https://developers.google.com/maps/documentation/android/start#get-key
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">YOUR_KEY_HERE</string>
```

Comment:
This file still got pushed up to GitHub because your `.gitignore` has `google_map_api.xml` instead of `google_maps_api.xml`